src/components/SettingsMembersBlock.vue

```diff
@@ -240,12 +240,12 @@ const errorMessage = computed(() => {
 
     <div class="mt-3">
       <div class="flex items-end gap-1">
-        <BaseInput
+        <TuneInput
```

Comment:
I don't like the design for error messages 🙈 previous one was nice :D 
Also, the alignment here looks weird
![Untitled](https://user-images.githubusercontent.com/15967809/233903007-f947e4e5-386c-4164-ab02-4873e6278594.png)

Previous: 
![image](https://user-images.githubusercontent.com/15967809/233903029-6c76e89a-6db7-49fe-afbf-d508a4cb1633.png)

Reply:
Cool cool :) can move the Author selection menu bit up?? 🙈 the alignment looks weird with the error message 😅

Reply:
Yep will fix that of course :)